apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Soeskenjustering.tsx

```diff
@@ -174,7 +172,14 @@ const Soeskenjustering = (props: SoeskenjusteringProps) => {
                 Legg til periode
               </NyPeriodeButton>
             ) : null}
-            <Button onClick={handleSubmit(ferdigstillForm)}>Lagre søskenjustering</Button>
+            <Button
+              type="submit"
+              onClick={(e) => {
+                handleSubmit(ferdigstillForm)(e)
+              }}
```

Comment:
```suggestion
              onClick={handleSubmit(ferdigstillForm)}
```
Burde ikke den fremdeles fungere?

Reply:
Enig, endrer tilbake.